### PR TITLE
Add new configuration token_expire_duration

### PR DESCRIPTION
**Pull request by josunect**

### CODE REVIEW

File: crd-docs/cr/kiali.io_v1alpha1_kiali.yaml [hunk 1]
```diff
@@ -423,6 +423,7 @@ spec:
     cache_namespaces:
     - ".*"
     cache_token_namespace_duration: 10
+    token_expire_duration: 60
```
Comment (jmazzitelli) [inline crd-docs/cr/kiali.io_v1alpha1_kiali.yaml:426]: See: https://github.com/kiali/kiali-operator/blob/master/DEVELOPING.adoc#are-you-altering-a-kiali-server-configuration-setting ... `Ensure each group of settings are alphabetically sorted in all the files where the new setting was added`

All the settings in all these files are sorted alphabetically. So you want to move this setting after the "qps" setting.

File: crd-docs/crd/kiali.io_kialis.yaml [hunk 2]
```diff
@@ -1126,6 +1126,9 @@ spec:
                   cache_token_namespace_duration:
                     description: "This Kiali cache is a list of namespaces per user. This is typically a short-lived cache compared with the duration of the namespace cache defined by the `cache_duration` setting. This is specified in seconds."
                     type: integer
+                  token_expire_duration:
+                    description: "This is the time, in seconds, that the service account token will be refreshed"
+                    type: integer
```
Comment (jmazzitelli) [inline crd-docs/crd/kiali.io_kialis.yaml:1131]: trivial - sort this, by moving below qps setting

Comment (jmazzitelli) [inline crd-docs/crd/kiali.io_kialis.yaml:1131]: trivial - sort this, by moving below qps setting

Comment (jmazzitelli) [reply crd-docs/crd/kiali.io_kialis.yaml:1131]: I wont' add this comment everywhere else - but you get the idea... this is trivial, but just make the settings sorted alphabetically.

File: crd-docs/cr/kiali.io_v1alpha1_kiali.yaml (hunk 1 above)
Comment (josunect) [reply crd-docs/cr/kiali.io_v1alpha1_kiali.yaml]: Changed

File: crd-docs/crd/kiali.io_kialis.yaml (hunk 2 above)
Comment (josunect) [reply crd-docs/crd/kiali.io_kialis.yaml]: Update this and the other changed files

**Comment by josunect** [comment]
TODO: add some tests to the Molecule config-values-test

**Review by jmazzitelli — APPROVED** [review]

**Comment by josunect** [comment]
Finally this change is not going to be longer needed. 
See https://github.com/kiali/kiali/pull/5386 for details

**Pull request closed without merging**